signals/signals.go

```diff
@@ -51,19 +52,19 @@ func (h *Handler) Loop() {
 	for {
 		select {
 		case <-h.quit:
-			h.log.Infof("=== Handler.Stop()'d ===")
+			level.Info(h.log).Log("msg", "=== Handler.Stop()'d ===")
 			return
 		case sig := <-sigs:
 			switch sig {
 			case syscall.SIGINT, syscall.SIGTERM:
-				h.log.Infof("=== received SIGINT/SIGTERM ===\n*** exiting")
+				level.Info(h.log).Log("msg", "=== received SIGINT/SIGTERM ===\n*** exiting")
 				for _, subsystem := range h.receivers {
 					_ = subsystem.Stop()
 				}
 				return
 			case syscall.SIGQUIT:
 				stacklen := runtime.Stack(buf, true)
-				h.log.Infof("=== received SIGQUIT ===\n*** goroutine dump...\n%s\n*** end", buf[:stacklen])
+				level.Info(h.log).Log("msg", "=== received SIGQUIT ===\n*** goroutine dump...\n%s\n*** end", buf[:stacklen])
```

Comment:
This is not correct. It's a sprintf (see `%s` in the message). You the lazy one here too.